src/main/java/seedu/address/model/client/RentalInformationContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.model.client;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Tests that a {@code Client}'s {@code Rental Information} matches any of the keywords given.
+ */
+public class RentalInformationContainsKeywordsPredicate implements Predicate<Client> {
+
```

Comment:
Should remove redundant newline if possible to maintain consistency. Same for line 27 and 31.

Reply:
Thanks for pointing that out!